securedrop/db.py

```diff
@@ -264,10 +264,18 @@ class Journalist(Base):
         "JournalistLoginAttempt",
         backref="journalist")
 
+    MIN_USERNAME_LEN = 8
```

Comment:
For the public record: the offline conversation says that there is no attack that is mitigated by longer usernames. Lowering to 3.